packages/loader/container-loader/src/containerContext.ts

```diff
@@ -50,6 +50,11 @@ import { Container } from "./container";
 
 const PackageNotFactoryError = "Code package does not implement IRuntimeFactory";
 
+/**
+ * The ContainerContext is the "permanent" part of a Container. The Container's runtime could change during a session
+ * while a client has the Container loaded (e.g. if a new code proposal is accepted), but the context would remain
+ * the same.
+ */
 export class ContainerContext implements IContainerContext {
```

Comment:
@ChumpChief I re-watched the video of the call where you went through the Container boot process with @ghoshkaj and myself and in that one you said that the ContainerContext could change when a code proposal is accepted, so this comment would be incorrect... but my quick attempt to figure out how (particularly, how the `contextChanged` event would trigger again) failed. The only paths that end up emitting that event seem to be the standard ways of creating a container (load, createDetached, rehydrate). Does an accepted code proposal cause the whole Container to be recreated? But then that would result in a contextChanged event from a new Container, not the original one.

The [docs on `IContainerContext`](https://github.com/alexvy86/FluidFramework/blob/container-docs/packages/common/container-definitions/src/runtime.ts#L115-L122) match what you said though... am I missing something? cc @markfields since that comment is yours so you might also have thoughts.

Reply:
Historically the context could swap out (as the IContainerContext docs indicate), but it doesn't really anymore.  It used to be that upon processing a code proposal we might reload the context, but that was removed a long time ago: https://github.com/microsoft/FluidFramework/pull/5725

So the IContainerContext docs are stale.